plugins/core/Standard/qHoughNormals/src/qHoughNormalsDialog.cpp

```diff
@@ -0,0 +1,62 @@
+//##########################################################################
+//#                                                                        #
+//#                       CLOUDCOMPARE PLUGIN: qHPR                        #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 or later of the License.      #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                  COPYRIGHT: Daniel Girardeau-Montaut                   #
+//#                                                                        #
+//##########################################################################
+
+#include "qHoughNormalsDialog.h"
+#include "ui_qHoughNormalsDlg.h"
+
+namespace {
+	#ifndef M_PI
```

Comment:
Can't we use the definitions in CCLib for Pi and DEG_TO_RAD / RAD_TO_DEG ?

Reply:
We could - I was just factoring it out from where it was inline before.

Doing that means we bring in:
```
#include "CCTypes.h"

//system
#include <cfloat>
#include <cmath>
#include <limits>
```

Which doesn't seem worth it to me. This isn't something that never changes (I hope!).